viz_scripts/energy_calculations.ipynb

```diff
@@ -237,172 +238,47 @@
-    "        filtered_notravel_data = data_eb[data_eb['Replaced_mode'] == \"No travel\"]\n",
-    "        filtered_freeshuttle_data = data_eb[data_eb['Replaced_mode'] == \"Free Shuttle\"]\n",
-    "        filtered_walk_data = data_eb[data_eb['Replaced_mode'] == \"Walk\"]\n",
-    "        filtered_GasCarShared_data = data_eb[data_eb['Replaced_mode'] == \"Gas Car Shared Ride\"]\n",
-    "\n",
-    "        selected_columns = ['distance', 'Replaced_mode_kg_CO2', 'Mode_confirm_kg_CO2', \"replaced_mode\", \"mode_confirm\"]\n",
-    "\n",
-    "        print(\"With Dynamic config mapping:\")\n",
-    "        print(\"\\n\")\n",
-    "\n",
-    "        print(\"Walk Data:\")\n",
-    "        print(str(filtered_walk_data[selected_columns].head()))\n",
-    "        print(\"\\n\")\n",
-    "\n",
-    "        print(\"No Travel Data:\")\n",
-    "        print(str(filtered_notravel_data[selected_columns].head()))\n",
-    "        print(\"\\n\")\n",
-    "\n",
-    "        print(\"Gas Car Shared Data:\")\n",
-    "        print(str(filtered_GasCarShared_data[selected_columns].head()))\n",
-    "        print(\"\\n\")\n",
-    "\n",
-    "        print(\"Free Shuttle Data:\")\n",
-    "        print(str(filtered_freeshuttle_data[selected_columns].head()))\n",
-    "        print(\"\\n\")\n",
-    "\n",
-    "        print(\"Bus Data:\")\n",
-    "        print(str(filtered_bus_data[selected_columns].head()))\n",
-    "        print(\"\\n\")\n",
-    "        \n",
-    "        print(\"Taxi/Uber/Lyft Data:\")\n",
-    "        print(str(filtered_taxi_data[selected_columns].head()))\n",
-    "        print(\"\\n\")\n",
-    "        \n",
-    "        print(\"Total Kg CO2 Emissions\")\n",
-    "        print(ebco2['total_kg_CO2_emissions'])\n",
+    "        if (is_debug_mode == True):\n",
+    "            ebco2_lb=data_eb.groupby('Replaced_mode').agg({'CO2_Impact(lb)': ['sum', 'mean']},)\n",
+    "            ebco2_lb.columns = ['total_lb_CO2_emissions', 'average_lb_CO2_emission']\n",
+    "            ebco2_lb = ebco2_lb.reset_index()\n",
+    "            ebco2_lb = ebco2_lb.sort_values(by=['total_lb_CO2_emissions'], ascending=False)\n",
+    "            \n",
+    "            scaffolding.print_CO2_emission_calculations(data_eb, ebco2_lb, ebco2, dynamic_labels)\n",
     "     \n",
     "    except:\n",
     "        generate_missing_plot(plot_title_no_quality,debug_df,file_name)\n",
     "        alt_text = store_alt_text_missing(debug_df, file_name, plot_title_no_quality)\n",
     "# case: when label_options is absent from the config file\n",
-    "# else: \n",
-    "#     plot_title_no_quality=f\"Sketch of Total pounds of CO2 Emissions of {mode_of_interest} trips\"\n",
-    "#     file_name =f'sketch_CO2impact_{mode_of_interest}%s' % file_suffix\n",
-    "\n",
-    "#     try:\n",
-    "#         ebco2=data_eb.groupby('Replaced_mode').agg({'CO2_Impact(lb)': ['sum', 'mean']},)\n",
```

Comment:
One of the "else" block was commented, while other was in place. This is because I wanted to observe the uniform kg_CO2 computation for CO2 emission, while observing the lb_CO2 computation for CO2 emission. These codes have been cleaned up, now we're displaying in Imperial system (i.e. lb, mile) for default mapping case, and Metric system (i.e. kg, km) case.